package-lock.json

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
Please don't commit `package-lock.json`, and add it to `.gitignore` instead. I think it's just asking for trouble to commit it, because it will inevitably get subtly out-of-sync with `yarn.lock` and then it will look like we're sanctioning two slightly different versions of dependencies. Hopefully [Yarn will support `package-lock.json` someday[(https://github.com/yarnpkg/yarn/issues/3614#issuecomment-316652699), then we can revisit this policy.

I should probably write this down in the practices repo.